src/style.css

```diff
@@ -1,7 +1,21 @@
-.hello {
-  color: rgb(58, 107, 180);
+.container {
+  margin: auto;
+  width: 30%;
+  display: flex;
+  justify-content: space-between;
 }
 
-body {
-  background-color: bisque;
+.scores-list {
+  padding: 0;
+  list-style: none;
+}
```

Comment:
- [ ] Kindly make sure the list has a `border-radius` around it 🙏 
- [ ] Kindly make sure each row in the list have alternating colors as seen in the Figma design 🙏 . You can use the CSS `:nth` selector for this. You can read more about it in this [article](https://css-tricks.com/almanac/selectors/n/nth-of-type/)

WireFrame Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/181936516-30c48b80-86f8-4f0b-bfc5-8b448b0052c3.png)|  ![image](https://user-images.githubusercontent.com/74814780/181936504-f17ac676-3156-4814-ae6c-c0bfa3551b9b.png)